admin_manual/configuration_server/occ_command.rst

```diff
@@ -108,40 +116,41 @@ optional, while commands are required. The syntax is::
 Get detailed information on individual commands with the ``help`` command, like 
 this example for the ``maintenance:mode`` command::
 
- $ sudo -u www-data php occ help maintenance:mode
-   Usage:
-   maintenance:mode [--on] [--off]
-
-   Options:
-   --on                  enable maintenance mode
-   --off                 disable maintenance mode
-   --help (-h)           Display this help message.
-   --quiet (-q)          Do not output any message.
-   --verbose (-v|vv|vvv) Increase the verbosity of messages: 1 for normal 
-   output, 2 for more verbose output and 3 for debug
-   --version (-V)        Display this application version.
-   --ansi                Force ANSI output.
-   --no-ansi             Disable ANSI output.
-   --no-interaction (-n) Do not ask any interactive question.
+ sudo -u www-data php occ help maintenance:mode
+ Usage:
+  maintenance:mode [options]
+
+ Options:
+      --on              enable maintenance mode
+      --off             disable maintenance mode
+  -h, --help            Display this help message
+  -q, --quiet           Do not output any message
+  -V, --version         Display this application version
+      --ansi            Force ANSI output
+      --no-ansi         Disable ANSI output
+  -n, --no-interaction  Do not ask any interactive question
+      --no-warnings     Skip global warnings, show command output only
+  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 
+                        2 for more verbose output and 3 for debug
 
 The ``status`` command from above has an option to define the output format.
 The default is plain text, but it can also be ``json``::
 
- $ sudo -u www-data php status --output =json
- {"installed":true,"version":"9.0.0","versionstring":"9.0",
- "edition":"Enterprise"}
+ sudo -u www-data php occ status --output=json
+ {"installed":true,"version":"9.0.0.19","versionstring":"9.0.0","edition":""}
 
 or ``json_pretty``::
 
- $ sudo -u www-data php status --output =json_pretty
+ root@studio:/var/www/owncloud# sudo -u www-data php occ status 
```

Comment:
@carlaschroder  In other command examples the command prompt is not shown, did you intent to put the "root@studio..." in here?